db/migrate/20220113213842_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, :likes_counter, :comment_counter
```

Comment:
- [ ] You can adjust this line as follows:
```suggestion
      t.references :author, foreign_key: { to_table: 'users' }
      t.integer :comments_counter
      t.integer :likes_counter
```

Reply:
Thanks, I have updated them.